### PR TITLE
(maint) Bump runtime to 201910020

### PR DESCRIPTION
**Pull request by rodjek**
https://jenkins-master-prod-1.delivery.puppetlabs.net/view/PDK/view/adhoc/job/platform_pdk_adhoc_pdk-van-build_adhoc/82/

### CODE REVIEW

File: configs/projects/pdk.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 project "pdk" do |proj|
   # Inherit a bunch of shared settings from pdk-runtime config
-  proj.setting(:pdk_runtime_version, '201907220')
+  proj.setting(:pdk_runtime_version, '201910020')
```
Comment (glennsarti) [inline configs/projects/pdk.rb]: Am I reading that right?  20 Oct 2019?

Comment (glennsarti) [reply configs/projects/pdk.rb]: Oh nope  Build 0 of 2 Oct 2019

File: configs/components/rubygem-minitar.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+component "rubygem-minitar" do |pkg, settings, platform|
+  pkg.version "0.6.1"
```
Comment (glennsarti) [inline configs/components/rubygem-minitar.rb]: will we need both 0.6.1. and 0.9 ?

File: configs/projects/pdk.rb (hunk 1 above)
Comment (rodjek) [reply configs/projects/pdk.rb]: 2019-07-02-0

File: configs/components/rubygem-minitar.rb (hunk 2 above)
Comment (rodjek) [reply configs/components/rubygem-minitar.rb]: Not for this release, but we will for the next.

**Comment by glennsarti** [comment]
Adhoc run is valid and correct.

**Pull request merged**